src/Components/VideoSearchApp.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import VideoSearchList from "./VideoSearchList";
+import Player from "./Player";
+import VideoSearch from "./VideoSearch";
+
+import { Container, Row, Col } from "react-bootstrap/";
+
+const VideoSearchApp = () => {
+  const [movies, setMovies] = useState([]);
+  const [movieId, setMovieId] = useState("");
+
+  const searchResultHandler = async (searchText) => {
+    const key = "AIzaSyBsGroGmofzjHBtIG_y4ugaJw8bqSp9eaY";
+    const response = await fetch(
+      "https://youtube.googleapis.com/youtube/v3/search?part=snippet&maxResults=25&q=" +

```

Comment:
Make this `key` and youtube API link, environment variables. Have a look [here](https://create-react-app.dev/docs/adding-custom-environment-variables/).